synchronizations/Synchronization.md

```diff
@@ -0,0 +1,223 @@
+            personRootModel.RootElements.Add(personRegister);
+
+            repository.Models.Add(new Uri("ttc:source"), familyRootModel);
+            repository.Models.Add(new Uri("ttc:target"), personRootModel);
+        }
+```
+
+Here, we are creating two instances of the [Model](../models/api/NMF.Models.Model.yml) class. These instances, `familyRootModel` and `personRootModel`, are initially empty. In our case, these models represent the root nodes in a tree-like structure.
+
+We then create instances of `FamilyRegister` and `PersonRegister`. These registers are child nodes of the models, and they manage specific types of data, in our case Familymembers and Persons:
+- `FamilyRegister`: As the name suggests, this register manages families. A family, in this context, consists of a parent couple (mother and father) and can have children.
+- `PersonRegister`: This register manages individual persons. These persons form the foundation for families, as each family is composed of individuals.
+
+
+```csharp
+synchronization.Synchronize(
+                synchronization.SynchronizationRule<Families2PersonsSynchronization.FamilyRegisterToPersonRegister>(), // synchronization block/rule
+                ref familyRegister, // Familyregister is the left model
+                ref personRegister, // Personregister is the right model
+                SynchronizationDirection.LeftWins, // prioritizes the left synchronization of left model (without shouldCorrespond it has no impact)
+                ChangePropagationMode.TwoWay); // TwoWay means Bidirectional synchronization
+```                
+Synchronization Method Call:
+    The `Synchronize()` method is invoked on the synchronization object to establish synchronization between two models.
+    This method configures how data is synchronized between the models.
+Synchronization Rule:
+    The synchronization rule, `FamilyRegisterToPersonRegister`, is specified from the `Families2PersonsSynchronization` class.
+    It defines the logic for mapping data between the family register and the person register.
+Adding Registers:
+    Both the family register and the person register are passed as parameters to the Synchronize method.
+    The family register is considered the left model, and the person register is considered the right model.
+Synchronization Direction:
+    The synchronization direction is set to LeftWins, indicating that the left model's values take precedence during synchronization.
+    This ensures that conflicts are resolved in favor of the left model's data.
+Change Propagation Mode:
+    The change propagation mode is set to TwoWay, indicating bidirectional synchronization.
+    Changes can propagate in both directions, allowing both models to react to each other's updates.
+
+
+```csharp
+familyRootModel.RootElements.Add(familyRegister);
+personRootModel.RootElements.Add(personRegister);
+
+repository.Models.Add(new Uri("ttc:source"), familyRootModel);
```

Comment:
The reason for Models.Add is that the model is registered but there is no attempt of serializing to disk, because it is unclear where the model would be stored. If you work with actual models, they would typically have file Uris, loading and saving the models through their Uris